book/history/sourdough-history.tex

```diff
@@ -100,11 +100,11 @@ \chapter{The history of sourdough}%
 Little did the people back then know that tiny microorganisms
 were the reason the bread was better. It is not clear when
 they started using a bit of the dough from the previous
-day for the next batch of dough. But by doing so, sourdough
-bread making was born: Wild yeast in the flour and in the air
-plus bacteria start to decompose the flour-water mixture, also
-known as your dough. The yeast makes the dough fluffy, and
-the bacteria primarily creates acidity. The different
+day for the next batch of dough. But by doing so, sourdough bread
```

Comment:
How about:

But by doing so, sourdough bread making---as we know it today---was born: Wild yeast
in the flour and in the air,  with bacteria starting to decompose the flour-water mixture.

Reply:
Great suggestion. Learned about the `---` today.